.project-template/refill_template_vars.sh

```diff
@@ -1,2 +1,2 @@
-TEMPLATE_DIR=$(dirname $(readlink -f "$0"))
+TEMPLATE_DIR=$(dirname "$0")
```

Comment:
We lose the traversing of symlinks and canonicalization here. I can't remember exactly why/when that needed to happen. I was surprised and disappointed that there seems to [not be any equivalent built in to Mac](https://stackoverflow.com/q/1055671/8412986)!

No objection to the change, since I can't think of a solution that's better on average...

Reply:
I know, such a bummer! Thanks for taking a look.